WMF Framework/CollectionViewEditController.swift

```diff
@@ -240,6 +279,13 @@ public class CollectionViewSwipeToEditController: NSObject, UIGestureRecognizerD
         }
     }
     
+    var areSwipeActionsDisabled: Bool = false {
+        didSet {
+            longPressGestureRecognizer.isEnabled = !areSwipeActionsDisabled
+            panGestureRecognizer.isEnabled = !areSwipeActionsDisabled
+        }
+    }
+    
     // MARK: - States
     
     func openActionPane(_ completion: @escaping (Bool) -> Void = {_ in }) {
```

Comment:
Naming question - how do we feel about implicit `completion` parameters? I think I prefer explicit vs "_", but I don't feel super strongly about it beyond maybe wanting consistency...

Reply:
It's fine unlabeled here as it's the first and only param. Autocomplete handles the case as well:
<img width="339" alt="screen shot 2018-02-08 at 7 36 57 am" src="https://user-images.githubusercontent.com/741327/35973293-df7fa256-0ca2-11e8-9339-5c4e4c8da144.png">
<img width="470" alt="screen shot 2018-02-08 at 7 36 13 am" src="https://user-images.githubusercontent.com/741327/35973294-df883b5a-0ca2-11e8-9644-87e73b3f8d96.png">